pkg/pipelines/bootstrap.go

```diff
@@ -157,6 +158,7 @@ func Bootstrap(o *BootstrapOptions, appFs afero.Fs) error {
 	if err != nil {
 		return fmt.Errorf("failed to build resources: %v", err)
 	}
+	log.Success("The default dev/stage environment have been created")
```

Comment:
How about "Created the following environments: cicd, dev, stage" but with the namespaces for these environments?